tests/MaterialDesignThemes.Wpf.Tests/Converters/SliderToolTipConverterTests.cs

```diff
@@ -11,7 +11,14 @@ public sealed class SliderToolTipConverterTests
     [InlineData(42)]
     public void SliderConverterTest(object value)
     {
+
+/* Unmerged change from project 'MaterialDesignThemes.Wpf.Tests (net8.0-windows)'
+Before:
         Wpf.Converters.SliderToolTipConverter converter = new ();
+After:
+        SliderToolTipConverter converter = new ();
+*/
+        Wpf.Converters.Internal.SliderToolTipConverter converter = new ();
```

Comment:
Not sure I understand this comment. Is it still needed?

```suggestion
        Wpf.Converters.Internal.SliderToolTipConverter converter = new ();
```